addon/templates/components/power-select-multiple/trigger.hbs

```diff
@@ -16,8 +16,8 @@
       {{/if}}
     </li>
   {{else}}
-    {{#if (and placeholder (not searchEnabled))}}
-      <span class="ember-power-select-placeholder">{{placeholder}}</span>
+    {{#if (not-eq placeholder maybePlaceholder)}}
```

Comment:
no you are right, i just realized this myself and am correcting it :) thx for the review